packages/solid/tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "./dist",
```

Comment:
the outDir in the extended tsconfig is `./dist` as well, might be redundant to add ?